src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -4,8 +4,12 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Security.Cryptography.X509Certificates</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NoWarn>CS3016</NoWarn>
```

Comment:
Taken from Pkcs code. Lot of the code disables this warning using #pragma, but not all. What is the preferred way?

Reply:
The .csproj way.  The pragmas are older.